data/account/participation.go

```diff
@@ -69,13 +69,18 @@ type participationIDData struct {
 }
 
 // ToBeHashed implements the Hashable interface.
-func (id *participationIDData) ToBeHashed() (protocol.HashID, []byte) {
+func (id *ParticipationKeyIdentity) ToBeHashed() (protocol.HashID, []byte) {
 	return protocol.ParticipationKeys, protocol.Encode(id)
 }
 
+// ToParticipationID creates a ParticipationID hash from the identity file.
+func (id ParticipationKeyIdentity) ToParticipationID() ParticipationID {
```

Comment:
I think that a method named `ID()` would be a better choice here, but I'm not overly particular here.